### ISSUE TITLE
Warning when running decentralised market for alpha 2

### ISSUE THREAD

**Issue by maaktweluit** [golemfactory/yagna#626]
On current master ( tested on 1f856b342b0286eab949ebdd955ba19571107703 )

When running ( taken from [release page](https://github.com/golemfactory/yagna/releases/tag/v0.3.4-mkt-dece-alpha.0) ):
```
cargo run --no-default-features --features market-decentralized --features gnt-driver -- service run --accept-terms
```

The logs are spammed, every 1 to 2 seconds, with this warning:
```
[2020-09-30T15:48:27Z WARN  ya_market_decentralized::protocol::discovery] Can't get Offers from [0x68be6b06ad56681f13ddd9aab907cdd39307284a]. Error: Failed to broadcast caused by gsb error: bad request: No service registered under given address '/public/market/protocol/mk1/discovery/offers/Get'..
```

Also the blender example does not get picked up:
```
YAGNA_APPKEY=<APP_KEY> python blender.py --subnet-tag devnet-alpha.2
```

**Comment by jiivan** [comment]
Might be related to #614

**Comment by maaktweluit** [comment]
seems you are correct :)

I have tested it again and the blender example does work now.
I suspect there where no nodes in the network with the same version i was running.

The warning still shows, but a lot less.

Now i do see an error related to this:
```
[2020-10-01T10:37:06Z ERROR ya_market_decentralized::rest_api::requestor::counter_proposal] Error occurred: Failed to send counter Proposal for Proposal [R-44d70dcca62243339e64b6e94c7eb5a78ed951f89a993866988982ff81099c4d]. Error: Countering Proposal [R-6c13c866c1d8bd85d5671e4ee9a0e35242201bbb411b3cf53e7a7115bc6a12a7] GSB error: bad request: No service registered under given address.
```

**Comment by mdtanrikulu** [comment]
Same as @maaktweluit, having this issue when running blender example over yajsapi. Plus Internal Server Error.
```js
[2020-10-01T13:49:28Z ERROR ya_market_decentralized::protocol::discovery::error::DiscoveryInitError] Error occurred: Failed to send counter Proposal for Proposal [R-0d99748a61ad3fa615733448bed65b3676e8531b8c251f5aadc7b8ed479df2e0]. Error: Countering Proposal [R-d898ceeb58d111a6a825c50e01a6e665611d4b10df6642ac199fc37a860a0690] GSB error: bad request: No service registered under given address..
[2020-10-01T13:49:28Z ERROR actix_http::response] Internal Server Error: Send(ProposalId { id: "0d99748a61ad3fa615733448bed65b3676e8531b8c251f5aadc7b8ed479df2e0", owner: Requestor }, Gsb(GsbProposalError("bad request: No service registered under given address", ProposalId { id: "d898ceeb58d111a6a825c50e01a6e665611d4b10df6642ac199fc37a860a0690", owner: Requestor })))
```

**Comment by nieznanysprawiciel** [comment]
#637

**Comment by tworec** [comment]
> Same as @maaktweluit, having this issue when running blender example over yajsapi. Plus Internal Server Error.
> 
> ```js
> [2020-10-01T13:49:28Z ERROR ya_market_decentralized::protocol::discovery::error::DiscoveryInitError] Error occurred: Failed to send counter Proposal for Proposal [R-0d99748a61ad3fa615733448bed65b3676e8531b8c251f5aadc7b8ed479df2e0]. Error: Countering Proposal [R-d898ceeb58d111a6a825c50e01a6e665611d4b10df6642ac199fc37a860a0690] GSB error: bad request: No service registered under given address..
> [2020-10-01T13:49:28Z ERROR actix_http::response] Internal Server Error: Send(ProposalId { id: "0d99748a61ad3fa615733448bed65b3676e8531b8c251f5aadc7b8ed479df2e0", owner: Requestor }, Gsb(GsbProposalError("bad request: No service registered under given address", ProposalId { id: "d898ceeb58d111a6a825c50e01a6e665611d4b10df6642ac199fc37a860a0690", owner: Requestor })))
> ```

@mdtanrikulu this is a separate issue, see here https://github.com/golemfactory/yagna/issues/651

**Comment by tworec** [comment]
Closed via #642 
It has not appeared after this fix, especially on today's testing sess
